monetio/models/_wrfchem_mm.py

```diff
@@ -88,6 +88,13 @@ def open_mfdataset(
         var_list.append("PSFC")
         # need to calculate surface pressure and dp and optionally dz here.
 
+        # Additional defaults for satellite analysis
+        var_list.append("PH")
+        var_list.append("PHB")
+        var_list.append("PB")
+        var_list.append("P")
+        var_list.append("T")
```

Comment:
Is this the same as "tk"?

Reply:
Unless this was pre - processed, it shouldn't be. In WRF, the variable T is the perturbation theta, with a base theta of 300 K.
To get the temperature in K, you need to do:
theta = 300 + ds["T"]
tk = theta * (pressure_in_pa / 100_000) ** (0.286), where 0.286 = R/Cp

As with the previous comment, I am not able to check this reader in detail now, but hopefully this helps.
Cheers
Pablo